app/jobs/delete_gbif_events_job.rb

```diff
@@ -3,11 +3,18 @@
 class DeleteGbifEventsJob < ApplicationJob
   queue_as :lupo_background
 
-  def perform(id, options = {})
-    event = Event.find_by(uuid: id)
+  def perform(ids, options = {})
+    label = options[:label]
+    index = options[:index]
 
-    event.destroy! if event.present?
+    # delete event records from mysql
+    sql = ActiveRecord::Base.sanitize_sql_array(["DELETE FROM events WHERE id IN (?)", ids])
+    ActiveRecord::Base.connection.execute(sql)
```

Comment:
When we delete the events using raw query, we are skipping the [`after_commit on: [:destroy]` ](https://github.com/datacite/lupo/blob/master/app/models/concerns/indexable.rb#L53)callback.

Reply:
@ashwinisukale yes this is by design as I will be using the bulk api to delete the documents in the events index.